assets/tests/components/searchForm.test.tsx

```diff
@@ -137,6 +160,30 @@ describe("SearchForm", () => {
     expect(testDispatch).toHaveBeenCalledWith(setSearchText(""))
   })
 
+  test("clicking the clear button also calls onClear when set", async () => {
+    const testDispatch = jest.fn()
+    const onClear = jest.fn()
+    const validSearch: SearchPageState = {
+      query: { text: "12", property: "run" },
+      isActive: false,
+      savedQueries: [],
+    }
+    const validSearchState = {
+      ...initialState,
+      searchPageState: validSearch,
+    }
+    const result = render(
+      <StateDispatchProvider state={validSearchState} dispatch={testDispatch}>
+        <SearchForm onClear={onClear} />
+      </StateDispatchProvider>
+    )
+
+    await userEvent.click(result.getByTitle("Clear"))
```

Comment:
note(non-blocking): based on this [paragraph about using `screen`](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen); Maybe we should replace usage of `const result` with `screen`?

I didn't check this, but maybe it'd look like this?
```suggestion
    render(
      <StateDispatchProvider state={validSearchState} dispatch={testDispatch}>
        <SearchForm onClear={onClear} />
      </StateDispatchProvider>
    )

    await userEvent.click(screen.getByTitle("Clear"))
```

Reply:
I'm sure this could apply to like, almost all our tests, and I'm not sure how much it's actually worth doing.

Reply:
Nice yeah I've been stuck in the habit of using `result` but can convert to using screen on new/revisited test cases! Looks like the only other change needed is to import `screen` at the top.

Reply:
Yeah it will apply to a lot of our tests (and I'm similarly used to just using `result`) but it sounds like that would be more in line with best practices.